pkg/controller/cluster/main-controller.go

```diff
@@ -720,6 +738,34 @@ func (c *Controller) checkAndCreateKESCSR(ctx context.Context, nsName types.Name
 	return nil
 }
 
+func (c *Controller) getCertIdentity(ns string, cert *miniov1.LocalCertificateReference) (string, error) {
+	var certbytes []byte
+	secret, _ := c.kubeClientSet.CoreV1().Secrets(ns).Get(context.Background(), cert.Name, metav1.GetOptions{})
```

Comment:
I think we should handle the error here, in case of an error secret will be `nil` and then `secret.Type` may crash

Reply:
Good catch, fixed now.